src/Particular.ThroughputCollector.Contracts/ThroughputSource.cs

```diff
@@ -1,6 +1,8 @@
 ﻿namespace Particular.ThroughputCollector.Contracts;
+
 public enum ThroughputSource
 {
+    None,
```

Comment:
Is this a valid option, if we create an` EndpointIndentifier`, what scenario would we ever select it?